src/compiler/checker.ts

```diff
@@ -981,6 +981,16 @@ namespace ts {
             return getExternalModuleMember(<ImportDeclaration>node.parent.parent.parent, node);
         }
 
+        function getTargetOfGlobalModuleExportDeclaration(node: GlobalModuleExportDeclaration): Symbol {
+            const moduleSymbol = node.parent.symbol;
+            if (moduleSymbol && moduleSymbol.exports && moduleSymbol.exports["export="]) {
+                return moduleSymbol.exports["export="].exportSymbol;

```

Comment:
Does this account for module augmentations? Will you need to get the "merged" symbol in that case?